src/routes/console/project-[project]/auth/+page.svelte

```diff
@@ -102,6 +105,21 @@
                                 </Pill>
                             </Copy>
                         </TableCell>
+                        <TableCell onlyDesktop title="Labels">
+                            {#each user.labels.slice(0, 2) as label, i}
+                                {#if i == 0}
+                                    <span
+                                        use:tooltip={{
+                                            disabled: user.labels.length < 2,
+                                            content: user.labels.join(', ')
+                                        }}>
+                                        <Pill>{label}</Pill>
+                                    </span>
+                                {:else}
+                                    +{user.labels.length - 1}
+                                {/if}
+                            {/each}
```

Comment:
@TorstenDittmann, any suggestion on how to make the pill and the additional text inline?